app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUiRenderer.kt

```diff
@@ -14,6 +14,8 @@ class CustomDrugEntryUiRenderer(
       return
     }
 
+    ui.hideProgressBar()
```

Comment:
Same comment as above, this can be moved to `if-else` condition and we can avoid using `return`